analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/ClearTextProtocolsAreSensitive.cs

```diff
@@ -61,16 +55,54 @@ public sealed class ClearTextProtocolsAreSensitive : HotspotDiagnosticAnalyzer
             {"clear-text SMTP", "SMTP over SSL/TLS or SMTP with STARTTLS" }
         };
 
+        private readonly List<string> commonlyUsedXmlDomains = new List<string>
```

Comment:
We'll never do `.Add` operation => we don't need a list. This can be an array. Same for others.